src/xcode/ENA/ENAUITests/HealthCertificate/ENAUITests_13_CreateHealthCertificate.swift

```diff
@@ -377,6 +378,27 @@ class ENAUITests_13_CreateHealthCertificate: CWATestCase {
 		XCTAssertEqual(app.cells.matching(identifier: AccessibilityIdentifiers.HealthCertificate.Person.certificateCell).count, 2)
 	}
 
+	func test_screenshot_HealthCertificate_FederalState_Flow() throws {
```

Comment:
```suggestion
	func test_HealthCertificate_FederalState_Flow() throws {
```

Seems like we don't take a screenshot